dev/data/data_common.py

```diff
@@ -23,27 +23,30 @@ def download_file(url: str, fname: str, chunk_size=1024):
             bar.update(size)
 
 
-def write_datafile(filename, toks):
+HEADERS_INFO = {
```

Comment:
i prefer to use dict of dicts, is more backwards compatible, flexible, and self-documenting. instead of tuples.

Reply:
done!